index.xml

```diff
@@ -175,6 +175,27 @@ of keys and assignment of `keyId` is out of scope for this document.
      </t>
     </section>
 
+    <section anchor="date" title="date">
+     <t>
+RECOMMENDED.  The `date` field is an integer representing a Unix time. Represent the
+signature issuing timestamp. A signature generated after the current timestamp
+should not be processed. Using Unix timestamp avoids speeds up processing and
+avoid timezone management existing in RFC3339. Subsecond precision is allowed
+using decimal notation.
```

Comment:
@liamdennehy replaced with `decimal notation`. Let me know if it's good for you. To me - considering network and reverse-proxy latency - integers are fine.

Reply:
Not sure why I was tagged, but ok :smile:

Reply:
I received a mail from gh with you asking `which float spec?` ;)

Reply:
Sorry, you did indeed :smile: I need to pay more attention